reflect_test.go

```diff
@@ -70,6 +70,9 @@ type TestUser struct {
 	Feeling ProtoEnum `json:"feeling,omitempty"`
 	Age     int       `json:"age" jsonschema:"minimum=18,maximum=120,exclusiveMaximum=true,exclusiveMinimum=true"`
 	Email   string    `json:"email" jsonschema:"format=email"`
+
+	SecretNumber int  `json:"secret_number,omitempty" jsonschema:"enum=9|30|28|52"`
+	Sex		string    `json:"sex,omitempty" jsonschema:"enum=male|female|neither|whatever|other|not applicable"`
```

Comment:
spacing looks off here - i think they used spaces in the original and this is a tab between `sex` and `string`

Reply:
Yeah, I don't have `gofmt` to run on this lib so I'll fix